yolo_results:
  ros__parameters:
    use_compressed_image: True
    show_image: True
    model_name: 'cotton_plant_model.engine'
    iou: 0.3
    conf: 0.3
